### PR TITLE
Feat/single video page

### PR DESCRIPTION
**Pull request by manish591**
#### Added Single Video Page

* Added react player to show youtube video
* Added notes form

[live preview](https://h9pmbl.csb.app/explore/watch/inVClSNYBQ4)

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
UI looks great
Code structure is good
PR is ready to merge

File: src/pages/video-page/VideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  return (
+    <div className="video-page">
+      <div className="video-page__main grid">
+        <div className="video-page__show-vodeo">
+          <ReactPlayer
+            url={`https://www.youtube.com/watch?v=${videoId}`}
+            controls
+            width="100%"
+            height="480px"
+          />
```
Comment (lomesshh) [inline src/pages/video-page/VideoPage.jsx:18]: got to know about react player !

File: src/pages/video-page/VideoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                className="flex"
+                style={{ alignItems: "center", opacity: "0.6" }}
+              >
+                <span class="material-icons-outlined">chat_bubble_outline</span>
+                <p>75 Comments</p>
+              </div>
+              <div
+                className="flex"
+                style={{ alignItems: "center", opacity: "0.6" }}
+              >
```
Comment (lomesshh) [inline src/pages/video-page/VideoPage.jsx:93]: avoid inline styling

**Review by lomesshh — APPROVED** [review]
UI Looks cool !

rest looks good to merge

File: src/components/notes/NotesCard.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+.notes-card {
+  background-color: rgba(139, 139, 139, 0.062);
+  block-size: 480px;
```
Comment (Rohanmond) [inline src/components/notes/NotesCard.css:3]: better to maintain same unit

**Review by Rohanmond — APPROVED** [review]
code is looks good to merge

File: src/pages/video-page/VideoPage.jsx (hunk 2 above)
Comment (manish591) [reply src/pages/video-page/VideoPage.jsx:93]: ok

**Pull request merged**